plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/controller/CodeTransformChatController.kt

```diff
@@ -457,6 +465,11 @@ class CodeTransformChatController(
         }
     }
 
+    private suspend fun handleDownloadFailed(failureReason: DownloadFailureReason) {
+        codeTransformChatHelper.addNewMessage(buildDownloadFailureChatContent(failureReason))
+        codeTransformChatHelper.addNewMessage(buildStartNewTransformFollowup())
```

Comment:
Not a blocker: How many failure functions call `buildStartNewTransformFollowup`? 6-7? 

Its out of scope for this PR, but would be a nice to have this abstracted so  any failure state that calls `buildStartNewTransformFollowup` goes through a single function.